tests/examples/pacer/nda/ca4_2.json

```diff
@@ -15,7 +15,8 @@
           "pacer_case_id": "164306",
           "pacer_doc_id": "00409006913",
           "pacer_magic_num": "55e8d970a5a88b7a",
-          "pacer_seq_no": null
+          "pacer_seq_no": null,
+          "short_description": "Notice re:"
```

Comment:
The `re:` here looks possibly wrong?

Reply:
Yeah, in this case, it certainly seems truncated:
![Screenshot 2023-04-21 at 17 02 49](https://user-images.githubusercontent.com/486004/233746698-a0373b9b-7a5f-4711-9bbd-bf813a951985.png)

The description in the footer is longer.
![Screenshot 2023-04-21 at 17 03 03](https://user-images.githubusercontent.com/486004/233746703-a8a9dce4-2a6a-42e6-b646-96c1a6480ec6.png)

It might be better to parse the short description from the footer for appellate notifications?

Reply:
That short description is quite long, but, assuming that there's an actual longer description somewhere, I think you're right that we should use the footer to get the whole thing.

Reply:
Yes, there is a longer one for the actual description:

`NOTICE re: Notice re: Regarding Resolution of Conflict with Proposed Argument dates by Bozzuto Management Company. [1001214590] [21-1919] Lynn Calkins`